package.json

```diff
@@ -55,11 +55,17 @@
     "@mui/x-date-pickers": "^5.0.15",
     "@netlify/plugin-nextjs": "^4.33.0",
     "@next/bundle-analyzer": "^10.2.3",
-    "@planet-sdk/common": "^0.1.11",
+    "@planet-sdk/common": "^0.1.15",
+    "@prisma/client": "^4.13.0",
     "@sentry/browser": "^6.15.0",
     "@sentry/integrations": "^6.19.2",
     "@sentry/node": "^6.19.2",
     "@sentry/webpack-plugin": "^1.18.3",
+    "@tanstack/react-query": "^4.29.5",
+    "@trpc/client": "^10.25.0",
+    "@trpc/next": "^10.25.0",
+    "@trpc/react-query": "^10.25.0",
+    "@trpc/server": "^10.25.0",
```

Comment:
Is using @prisma and @trpc possible with our Netlify and Heroku deployments of planet-webapp?

Reply:
The aim is to move to a single deployment on Vercel (#1877 will come first)

Reply:
So I guess we need to disable automatic deployment of the production branch main for Netlify and Heroku once we want to merge this in order to not break that deployments until there got migrated.